### PR TITLE
.NET 6.0

### PR DESCRIPTION
**Pull request by erikzhang**
Related to neo-project/neo#2625
Fix https://github.com/neo-project/neo-modules/issues/674

### CODE REVIEW

File: src/OracleService/OracleService.cs [hunk 1]
```diff
@@ -225,18 +229,8 @@ private static async Task SendContentAsync(Uri url, string content)
...
-                HttpWebRequest request = (HttpWebRequest)WebRequest.Create(url);
-                request.Method = "POST";
-                request.ContentType = "application/json";
-                request.Timeout = 5000;
-                using (StreamWriter dataStream = new StreamWriter(await request.GetRequestStreamAsync()))
-                {
-                    await dataStream.WriteAsync(content);
-                }
-                HttpWebResponse response = (HttpWebResponse)await request.GetResponseAsync();
-                if (response.ContentLength > ushort.MaxValue) throw new Exception("The response it's bigger than allowed");
```
Comment (shargon) [inline src/OracleService/OracleService.cs:237]: We will lose this condition?

Comment (erikzhang) [reply src/OracleService/OracleService.cs:237]: But we don’t care about response at all, do we?

Comment (shargon) [reply src/OracleService/OracleService.cs:237]: Is the download time included in the timeout ?

**Review by shargon — APPROVED** [review]

File: src/OracleService/OracleService.cs [hunk 2]
```diff
@@ -30,6 +29,11 @@ public class OracleService : Plugin, IPersistencePlugin
     {
         private const int RefreshIntervalMilliSeconds = 1000 * 60 * 3;
 
+        private static readonly HttpClient httpClient = new()
+        {
+            Timeout = TimeSpan.FromSeconds(5)
```
[the remark this answers is not in the document]
Comment (erikzhang) [reply src/OracleService/OracleService.cs]: Can you be sure that all responses are short than `ushort.MaxValue`?

[the remark this answers is not in the document]
Comment (shargon) [reply src/OracleService/OracleService.cs]: We should set a max response size, maybe 1mb

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -6,7 +6,7 @@ on:
   pull_request:
 
 env:
-  DOTNET_VERSION: 5.0.100
+  DOTNET_VERSION: 6.0.x
```
Comment (superboyiii) [inline .github/workflows/main.yml:9]: Why not?
```suggestion
  DOTNET_VERSION: 6.0.100
```

**Review by superboyiii — APPROVED** [review]

**Comment by erikzhang** [comment]
Wait for neo-node.

**Comment by erikzhang** [comment]
Why `NeoFS.API` uses CI version of packages?

**Review by erikzhang — COMMENTED** [review]
Why NeoFS.API uses CI version of packages?

File: src/OracleService/OracleService.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (erikzhang) [reply src/OracleService/OracleService.cs]: Maybe we need some tests?

**Comment by ZhangTao1596** [comment]
> Why NeoFS.API uses CI version of packages?

There will be problems if `NeoFS.API` use `v3.0.3` and neo-modules use `v3.0.3-CI01319`. So I updated it.

File: src/OracleService/OracleService.csproj [hunk 4]
```diff
@@ -5,7 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Neo.ConsoleService" Version="1.0.0" />
+    <PackageReference Include="Neo.ConsoleService" Version="1.2.0" />
     <PackageReference Include="NeoFS.API" Version="3.0.3" />
```
Comment (ZhangTao1596) [inline src/OracleService/OracleService.csproj:9]:
```suggestion
    <PackageReference Include="NeoFS.API" Version="3.1.0" />
```

It has been updated to `v3.1.0`.

**Comment by erikzhang** [comment]
> There will be problems if NeoFS.API uses v3.0.3 and neo-modules uses v3.0.3-CI01319. So I updated it.

We should never use CI packages in master branch.

**Review by vncoelho — COMMENTED** [review]
For now it is all working for NeoCompiler Eco.

**Review by shargon — APPROVED** [review]

**Review by superboyiii — APPROVED** [review]

**Review by Liaojinghui — APPROVED** [review]

**Pull request merged**